common/test/db_fixtures/certificates_web_view.json

```diff
@@ -106,9 +106,7 @@
       "change_date": "2050-06-15 11:02:13.007790+00:00",
       "changed_by": 99,
       "enabled": true,
-      "dashboard_tracking_code": "edx-course-v1&TESTCOURSE",
-      "company_identifier": "7nTFLiuDkkQkdELSpruCwD4F6jzqtTFsx3PfJUIT2qHqXRLG1",
-      "trk_partner_name": "edx"
+      "company_identifier": 1337
```

Comment:
Integer type, not string?

Is this change supposed to be in other PR but just thrown in here?

Reply:
From what I can tell, the company identifiers are just integers and this json supports it.

And I actually think this change needs to come when the model itself is changing. I had it in the other PR but it broke some configuration stuff so here we are.